pipenv/core.py

```diff
@@ -826,6 +826,8 @@ def do_install_dependencies(
     deps_list = list(lockfile.get_requirements(dev=dev, only=requirements))
     if requirements:
         index_args = prepare_pip_source_args(project.sources)
+        if pypi_mirror:
+            index_args.extend(["-i", pypi_mirror])
```

Comment:
This wouldn't be valid if there were multiple indexes available, since pip expects to receive only one index using `-i`. We have a method for implementing the `pypi_mirror` already by merging it into the project sources, which you can find in the `utils` module and you can use like this:

```python
index_args = prepare_pip_source_args(get_source_list(pypi_mirror=pypi_mirror, project=project))
```

Reply:
Fixed it